Server/src/app.ts

```diff
@@ -1,5 +1,5 @@
 import 'dotenv/config';
-const loadStartupProcess =  require('./loaders/loadStartupProcess');
+import { loadStartupProcess } from './loaders/loadStartupProcess'
```

Comment:
i like this es6 styling better then native mode. We should maybe add esilnt to backend to detect these for us.

Reply:
I'll agree that this looks much nicer. As it is the import statements in models are nearly identical to this; standardizing would make a whole lot of sense